projectDocs/Design Implementation

```diff
@@ -0,0 +1,35 @@
+# Design (Subject to updates)
+- All uses base view from submodule
+- Student/Staff submit new issue
+  - login is optional, login with D&E info else anonymous
+    - If anonymous then ask for contact details?
+  - Restful API to database 
+    - Database will have: 
+      - pk
+      - subject
+      - Issue content
```

Comment:
I don't think this is needed. Following GitHub, the "description" of the Issue is actually a connected Comment-- a Comment is automatically created along with an Issue being created.